client/src/App.js

```diff
@@ -1,10 +1,12 @@
 import React, { Fragment } from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import LandingPage from './components/layouts/LandingPage';
+import NavBar from './components/commonComponents/navBar/index';
```

Comment:
Putting index in the path is unnecessary here. :information_desk_person:

Reply:
ok thanks 💁